src/atoms/DropDown/index.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import FieldContainer from './DefaultFieldContainer'
+import TogglableArrowIcon from './DefaultIcon'
+import MenuContainer from './DefaultMenuContainer'
+import Option from './DefaultOption'
+
+import Relative from './../Relative'
+import Text from './../Text'
+
+const DEFAULT_PLACEHOLDER_STRING = 'Select'
+
+class Dropdown extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      selected: props.value || {
+        label: typeof props.placeholder === 'undefined' ? DEFAULT_PLACEHOLDER_STRING : props.placeholder,
+        value: ''
+      },
+      isOpen: false
+    }
+    this.mounted = true
+  }
+
+  componentWillReceiveProps (newProps) {
+    if (newProps.value && newProps.value !== this.state.selected) {
+      this.setState({selected: newProps.value})
+    } else if (!newProps.value) {
+      this.setState({selected: {
+        label: typeof newProps.placeholder === 'undefined' ? DEFAULT_PLACEHOLDER_STRING : newProps.placeholder,
```

Comment:
Can this be written as newProps.placeholder || DEFAULT_PLACEHOLDER_STRING ?

Reply:
No. It can't be. If I dont want any placeholder, then I should be able to set `''` as placeholder. According to your code, we can't set `''` as placeholder.